static/usr/share/subiquity/console-conf-wrapper

```diff
@@ -0,0 +1,109 @@
+#!/bin/bash
+set -e
+
+trap true HUP INT QUIT TSTP
+echo_on () { stty echo; }
+trap echo_on EXIT
+
+# agetty only sets ICRNL if it has read the username and seen whether
+# it was terminated by CR or NL. We pass -n to agetty so that hasn't
+# happened and need to force it on. Yay UNIX!
+stty icrnl -echo
+
+if [ -e /run/snapd-recovery-chooser-triggered ]; then
+    # make sure the chooser binary is there to avoid getting into an awkward
+    # state and locking out the user
+    if [ -x /usr/lib/snapd/snap-recovery-chooser ]; then
+        # when recovery chooser is invoked it takes over the terminal
+        exec /usr/lib/snapd/snap-recovery-chooser
+    fi
+    if [ -e /var/lib/console-conf/complete ]; then
+        exit 0
+    fi
+fi
+
+# always prefer to use modeevn if it is available
+if [ -e "/var/lib/snapd/modeenv" ]; then
+    mode="$(sed -n 's/mode=\([^[:space:]]*\)/\1/p' /var/lib/snapd/modeenv)"
+else
+    mode="$(sed 's/.*snapd_recovery_mode=\([^[:space:]]*\)[[:space:]].*/\1/' /proc/cmdline)"
```

Comment:
And `/usr/libexec/core/get-arg` for this one.

Reply:
done too